v4-client-js/src/clients/native.ts

```diff
@@ -147,6 +147,12 @@ export async function connectWallet(mnemonic: string): Promise<string> {
     globalThis.wallet = await LocalWallet.fromMnemonic(mnemonic, BECH32_PREFIX);
     globalThis.nobleWallet = await LocalWallet.fromMnemonic(mnemonic, NOBLE_BECH32_PREFIX);
 
+    const { privateKey, publicKey } = deriveHDKeyFromMnemonic(mnemonic);
```

Comment:
we are already setting the hdKey in `deriveMnemomicFromEthereumSignature`, are we able to use it?

This way we shouldn't need to make `deriveHDKeyFromMnemonic`

Reply:
ohhh ok TIL